wire-ios/Wire-iOS UnitTests/UserInterface/Conversation/Content/Cells/MLSMigrationCellDescriptionTests.swift

```diff
@@ -47,4 +47,94 @@ final class MLSMigrationCellDescriptionTests: XCTestCase {
         XCTAssertNil(cellDescription.accessibilityIdentifier)
         XCTAssertEqual(cellDescription.accessibilityLabel?.isEmpty, false)
     }
+
+    // MARK: - Attributed Strings
+
+    func test_mlsMigrationStarted_doesContainLinkInAttributedString() throws {
+        // given
+        let cellDescription = MLSMigrationCellDescription(messageType: .mlsMigrationStarted)
+        let expectation = self.expectation(description: "")
+
+        // when
+        let attributedString = try XCTUnwrap(cellDescription.configuration.attributedText)
+
+        // then
+        attributedString.enumerateAttribute(.link, in: attributedString.wholeRange, using: { value, _, processPtr in
+            guard value != nil else { return }
+            expectation.fulfill()
+            processPtr.pointee = false
+        })
+
+        waitForExpectations(timeout: 0.1)
+    }
+
+    func test_mlsMigrationFinalized_doesContainLinkInAttributedString() throws {
+        // given
+        let cellDescription = MLSMigrationCellDescription(messageType: .mlsMigrationFinalized)
+        let expectation = self.expectation(description: "")
+
+        // when
+        let attributedString = try XCTUnwrap(cellDescription.configuration.attributedText)
+
+        // then
+        attributedString.enumerateAttribute(.link, in: attributedString.wholeRange, using: { value, _, processPtr in
+            guard value != nil else { return }
+            expectation.fulfill()
+            processPtr.pointee = false
+        })
+
+        waitForExpectations(timeout: 0.1)
+    }
+
+    func test_mlsMigrationOngoingCall_doesContainLinkInAttributedString() throws {
```

Comment:
is there a typo in the name? Should it be `doesntContain` instead? Since the expectation is only fulfilled if the value is nil